js-variables-and-numbers/js-variables-and-numbers_social-media-post/js/index.js

```diff
@@ -10,13 +10,21 @@ console.clear();
 */
 
 // --v-- write your code here --v--
-
+const title = "Social Media Post";
+const text = "generic text";
+let likes = 20;
+const userId = "Juan Diaz";
+const isReported = true;
 // --^-- write your code here --^--
 
 /*
 2. Log all variables to the console. Increase the likes by one and log the amount of likes again. Adapt your code from 1.1 if necessary.
 */
 
 // --v-- write your code here --v--
-
+console.log(title);
+console.log(text);
+console.log(likes + 1);
```

Comment:
So wäre glaube ich richtig.

console.log("Title:", title);
console.log("Text:", text);
console.log("Likes:", likes);
console.log("User:", userId);
console.log("Is Reported:", isReported);

const newLike = likes + 1;
console.log("Updated Likes:", newLike);